modules/migration-mtc-release-notes-1-8.adoc

```diff
@@ -0,0 +1,73 @@
+
+[id="resolved-issues-1-8_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+.Indirect migration is stuck on backup stage
+
+In previous releases, indirect migration becomes stuck at the backup stage, due to invalid image name error.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2233097[*BZ#2233097*])
+
+.PodVolumeRestore remain In Progress keeping the migration stuck at Stage Restore
+
+In previous releases, on performing any indirect migration, the migration becomes stuck at the `Stage Restore` step, waiting for the `podvolumerestore` to be completed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2233868[*BZ#2233868*])
+
+.Migrated application unable to pull image from internal registry on target cluster
+
+In previous releases, on migrating an application to the target cluster, the migrated application fails to pull the image from the internal image registry resulting in an `application failure`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2233103[*BZ#2233103*])
+
+.Migration failing on Azure due to authorization issue
+
+In previous releases, on trying to perform a migration on an Azure cluster, backing up to Azure storage. The migration fails at the `Backup` stage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2238974[*BZ#2238974*])
+
+[id="known-issues-1-8_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Old restic pods are not getting removed on upgrading MTC 1.7.x -> 1.8.x
+
+In this release, on upgrading the MTC operator from 1.7.x to 1.8.x, the old Restic pods are not being removed. Hence, after the upgrade, both Restic and node-agent pods are visible in the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2236829[*BZ#2236829*])
+
+.Migrated builder pod fails to push to image registry
+
+In this release, on migrating an application including a `BuildConfig` from a source to target cluster, builder pod results in `error`, failing to push the image to the image registry. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2234781[*BZ#2234781*])
+
+.[UI] CA bundle file field is not properly cleared
+
+In this release, after enabling `Require SSL verification` and adding content to the CA bundle file for an MCG NooBaa bucket in MigStorage, the connection fails as expected. However, when reverting these changes by removing the CA bundle content and unchecking `Require SSL verification`, the connection still fails. The issue is only resolved by deleting and re-adding the repository. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2240052[*BZ#2240052*])
+
+
+[id="technical-changes-1-8_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+* Migration of {product-title} 3 to {product-title} 4 requires a legacy {mtc-full} ({mtc-short}) Operator and MTC 1.7.x.
+* It is not supported to migrate MTC 1.7.x to MTC 1.8.x.
+* You must use MTC 1.7.x to migrate anything with a source of {product-title} <4.10.
```

Comment:
Suggestions: "Migration of containers from MTC 1.7.x to MTC 1.8.x is not supported." or "Upgrading from MTC 1.7.x to MTC 1.8.x is not supported."  -- depending which you mean.